### PR TITLE
pourRNA recipe

### PR DESCRIPTION
**Pull request by entzian**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pourRNA/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+./configure --prefix=$PREFIX
```
Comment (j-4) [inline recipes/pourRNA/build.sh]: Can you please do a "${PREFIX}" to be sure, that nothing is messed up with spaces etc

File: recipes/pourRNA/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+test:
+  commands:
+    - pourRNA --version
+
+about:
+  home: https://github.com/ViennaRNA/pourRNA/
+  license: GPL
+  license_file: COPYING
+  summary: "Compute local minima and energy barriers of a landscape."
+
```
Comment (j-4) [inline recipes/pourRNA/meta.yaml]: Is this tool published?
If so, you could add something like:
```
extra:
  identifiers:
- doi: xx/xxx.xx.xxx
```

**Comment by bgruening** [comment]
@entzian @j-4 @eggzilla question for the viennarna experts. Would it make sense to include a `run_exports` into the viennarna package (https://docs.conda.io/projects/conda-build/en/latest/source/define-metadata.html?highlight=run_exports#export-runtime-requirements). This would ensure that the correct viennarna package is automatically added as run-time dependency.
If not we probably need to pin viennarna here?

**Comment by j-4** [comment]
`run_exports` sounds like a valid thing to do here! viennarna acually is just a shared libarary with the swig python wrapper, so I guess this is an ideal use case for that!

**Comment by eggzilla** [comment]
I am always pinning viennarna in my recipes, gives the package authors more freedom to make
breaking API changes.

File: recipes/pourRNA/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - viennarna
+  run:
+    - viennarna
+test:
+  commands:
+    - pourRNA --version
+
+about:
+  home: https://github.com/ViennaRNA/pourRNA/
+  license: GPL
```
Comment (eggzilla) [inline recipes/pourRNA/meta.yaml]: Please specify the GPL version

**Comment by bgruening** [comment]
Pinning is good if there are ABI changes, but keep in mind that this makes it impossible to mix environments (if tools have different VRNA pinnings). So the pinning should be ideally as loose as possible. Is there any guarantee from Ronny to only break ABI/API in x. or x.x. releases?

**Comment by j-4** [comment]
Yes, Ronny only breaks ABI on major releases, so earliest with verions 3.0 of the package!

File: recipes/pourRNA/meta.yaml (hunk 2 above)
Comment (entzian) [reply recipes/pourRNA/meta.yaml]: It is submitted. I will add it in the next version.

**Comment by bgruening** [comment]
@j-4 your call :)
I would pin viennarna here to <3 for the time being and include run_exports in viennarna at some point.

Thanks @entzian !

**Comment by entzian** [comment]
This should be correct: >=2.4.11,<3.0.0. I don't know why lint fails.

**Comment by martin-raden** [comment]
@entzian you have to rebase your branch to the current state of the master branch to solve the linting issue (it is not about your package but some other)

**Comment by bgruening** [comment]
Merci!!!

**Pull request merged**